tools/stylelint-rules/comment-no-loud.js

```diff
@@ -0,0 +1,42 @@
+// It's fun to write JavaScript
+const stylelint = require('stylelint');
+
+const ruleName = 'spartacus/comment-no-loud';
+const messages = stylelint.utils.ruleMessages(ruleName, {
+  expected: 'Expected // for comments instead of /*',
+});
+
+module.exports = stylelint.createPlugin(ruleName, function () {
+  return function (root, result) {
+    const validOptions = stylelint.utils.validateOptions(result, ruleName, {
+      // No options
+    });
+
+    if (!validOptions) {
+      return;
+    }
+
+    root.walkComments((comment) => {
+      if (isLoudComment(comment)) {
+        stylelint.utils.report({
+          message: messages.expected,
+          node: comment,
+          result,
+          ruleName,
+        });
+      }
+    });
+  };
+});
+
+function isLoudComment(comment) {
+  const regex = new RegExp(/^[ \t\n]*\/\*(?!\!)/);
+
+  const splitComment = comment.source.input.css.split('\n');
```

Comment:
I'm no familair with this syntax, but is the source.input.css the way to only do this on (s) css?